snapcraft/plugins/catkin.py

```diff
@@ -87,6 +87,8 @@
 from snapcraft import common, file_utils, formatting_utils, repo
 from snapcraft.internal import errors, mangling
 
+from snapcraft.project import Project
```

Comment:
Please make this import conditional to typing

Reply:
Done in 9fe00f9